src/components/ExploreDashboardMenu/index.js

```diff
@@ -0,0 +1,70 @@
+import { Typography } from "@material-ui/core";
+import { TreeItem, TreeView } from "@material-ui/lab";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import { ReactComponent as CheckIcon } from "@/pesayetu/assets/icons/checked.svg";
+import Link from "@/pesayetu/components/Link";
+
+const ExploreDashboardMenu = ({ items, ...props }) => {
+  const classes = useStyles(props);
+  if (!items || !items.length) {
+    return null;
+  }
+
+  return (
+    <div className={classes.root}>
+      {items.map(({ children, label, path }) => (
+        <>
```

Comment:
What does this do?

Reply:
this renders the children @kilemensi

Reply:
Focus @KobbyMmo: **this** meant `<>` and what does it do meant why do we need it?